test/specs/integration.spec.js

```diff
@@ -50,6 +50,7 @@ describe('Integration tests', function() {
   beforeEach(function(done) {
     // Make sure we're connected to Firebase.  This can take a while on slow
     // connections.
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 180000;
```

Comment:
I don't want to set this too high, since when there is a real bug causing an async test not to finish, it'll now take 3 minutes for the test to fail!  Were you able to reproduce the timeouts yourself?  I'm fine with e.g. doubling the timeout (not sure what the default is though) if we think it'll make the tests more reliable, but 3 minutes is probably too far. :-)